### PR TITLE
[TE] monitoring - add alert and detection task counters

### PR DESCRIPTION
**Pull request by apucher**

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/detection/algorithm/MovingWindowAlgorithmTest.java [hunk 1]
```diff
@@ -21,9 +21,6 @@
 import java.util.List;
```
Comment (jihaozh) [inline thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/detection/algorithm/MovingWindowAlgorithmTest.java]: Looks like a unrelated file change.

Comment (apucher) [reply thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/detection/algorithm/MovingWindowAlgorithmTest.java]: Good catch. I wonder if it's worth making a separate PR though

Comment (jihaozh) [reply thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/detection/algorithm/MovingWindowAlgorithmTest.java]: Let's just put it here. Thank you for cleaning up the tests!
